v3/pforzheim.py

```diff
@@ -0,0 +1,113 @@
+    opening_hours: str = StringValidator(max_length=255, multiline=True)
+
+
+class PforzheimConverter(JsonConverter):
+    pforzheim_row_validator = DataclassValidator(PforzheimRowInput)
+
+    source_info = SourceInfo(
+        id='pforzheim',
+        name='Stadt Pforzheim',
+        public_url='https://www.pforzheim.de',
+    )
+
+    type_mapping: dict[str, ParkingSiteTypeInput] = {
+        'onStreet': ParkingSiteTypeInput.ON_STREET,
+        'carPark': ParkingSiteTypeInput.CAR_PARK,
+        'undergroundCarPark': ParkingSiteTypeInput.UNDERGROUND,
+    }
+
+    def handle_json(self, data: dict | list) -> ImportSourceResult:
+        import_source_result = self.generate_import_source_result(
+            static_parking_site_inputs=[],
+            static_parking_site_errors=[],
+        )
+
+        # We start at row 2, as the first one is our header
+        for item in data:
+            input_dict = {
+                'uid': item.get('Id'),
+                'name': item.get('name'),
+                'lat': str(item.get('lat')),
+                'lon': str(item.get('lon')),
+                'operator_name': item.get('operatorID'),
+                'address': item.get('address'),
+                'description': item.get('description'),
+                'type': item.get('type'),
+                'capacity_woman': item.get('quantitySpacesReservedForWomen'),
+                'capacity_disabled': item.get('quantitySpacesReservedForMobilityImpededPerson'),
+                'is_supervised': True if 'ja' in item.get('securityInformation').lower() else False,
+                'fee_description': item.get('feeInformation'),
+                'capacity': item.get('capacity'),
+                'opening_hours': item.get('openingHours'),
+            }
+
+            try:
+                input_data: PforzheimRowInput = self.pforzheim_row_validator.validate(input_dict)
+            except ValidationError as e:
+                import_source_result.static_parking_site_errors.append(
+                    ImportParkingSiteException(
+                        uid=input_dict.get('id'),
+                        message=f'validation error for {input_dict}: {e.to_dict()}',
+                    ),
+                )
+                continue
+
+            parking_site_input = StaticParkingSiteInput(
+                uid=name_to_id(self.source_info.id, input_data.name) if input_data.uid == '' else input_data.uid,
+                name=input_data.name,
+                type=self.type_mapping.get(input_data.type),
+                lat=input_data.lat,
+                lon=input_data.lon,
+                address=input_data.address.replace('\n', ' '),
+                description=input_data.description,
+                capacity=input_data.capacity,
+                capacity_woman=input_data.capacity_woman,
+                capacity_disabled=input_data.capacity_disabled,
+                fee_description=input_data.fee_description,
```

Comment:
At the moment, the park-api service does not allow newlines in descriptions, so you will have to remove them here.